src/System.Data.Common/tests/System/Data/DataColumnTest2.cs

```diff
@@ -436,31 +385,24 @@ public void ctor_ByColumnNameType()
         {
             Type typTest;
             DataColumn dc = null;
-            string[] sTypeArr = { "System.Boolean", "System.Byte", "System.Char", "System.DateTime",
-                "System.Decimal", "System.Double", "System.Int16", "System.Int32",
-                "System.Int64", "System.SByte", "System.Single", "System.String",
-                "System.TimeSpan", "System.UInt16", "System.UInt32", "System.UInt64" };
+            Type[] typeArr = { typeof(bool), typeof(byte), typeof(char), typeof(DateTime),
+                typeof(decimal), typeof(double), typeof(short), typeof(int),
+                typeof(long), typeof(sbyte), typeof(float), typeof(string),
+                typeof(TimeSpan), typeof(ushort), typeof(uint), typeof(ulong) };
 
-            foreach (string sType in sTypeArr)
+            foreach (Type type in typeArr)
             {
-                typTest = Type.GetType(sType);
-                dc = new DataColumn("ColName", typTest);
-                // ctor - object
-                Assert.False(dc == null);
+                dc = new DataColumn("ColName", type);
 
                 // ctor - ColName
-                Assert.Equal(typTest, dc.DataType);
+                Assert.Equal(type, dc.DataType);
             }
         }
 
         [Fact]
         public void ctor_ByColumnNameTypeExpression()
         {
-            DataColumn dc;
-            dc = new DataColumn("ColName", typeof(string), "Price * 1.18");
-
-            // ctor - object
-            Assert.False(dc == null);
+            _ = new DataColumn("ColName", typeof(string), "Price * 1.18");
```

Comment:
Anything to validate with asserts instead?  e.g. that the values passed in are available through some properties in some way?